testing_cli.py

```diff
@@ -1,202 +1,235 @@
 """Provides a continuous command line prompt for testing drone capabilities."""
-
 import argparse
+import sys
 import threading
+import traceback
 
+import config
 from flight.drone.drone_controller import DroneController
-from flight import constants as c
+from flight import constants
+from flight.tasks import TaskFactory
+from flight.tasks.encodings import Encodings
 
 PROMPT_FOR_COMMAND = '> '
 
+CLI_STR_TO_TASK = {
```

Comment:
All of this should definitely be defined in a grammar somewhere else, a system similar to my attempt in networkflight could solve this since each class was named. The fact that we have multiple definitions mapping x to y and y to x is wasteful and confusing.

Again, I think you were trying to reinvent the wheel